src/drive/locales/en.json

```diff
@@ -462,9 +462,13 @@
   "Viewer": {
     "close": "Close",
     "noviewer": {
-      "title": "The viewer doesn't know how to read this type of file yet.",
       "download": "Download this file",
-      "openWith": "Open with..."
+      "openWith": "Open with...",
+      "cta": {
+        "saveTime": "Earn some time!",
```

Comment:
I know the card says `Earn`, but really it's `Save`. You can earn money, but not time AFAIK.